test/volapi.test.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2016, Joyent, Inc.
+ */
+
+var VOLAPI = require('../lib/index').VOLAPI;
+var bunyan = require('bunyan');
+var restify = require('restify');
+
+var VOLAPI_IP = process.env.VOLAPI_IP || '10.99.99.70';
+var VOLAPI_URL = 'http://' + VOLAPI_IP;
+
+var volApiClient;
+
+exports.setUp = function (callback) {
+    var logger = new bunyan.createLogger({
+            name: 'vapi_unit_test',
```

Comment:
s/vapi/volapi/

Call the var `log` instead of `logger` perhaps? Suspect you just copied other test files here tho.